tests/functional/api/test_projects.py

```diff
@@ -333,13 +334,17 @@ def test_project_groups_list(gl, group):
 
 def test_project_transfer(gl, project, group):
     assert project.namespace["path"] != group.full_path
-    project.transfer_project(group.id)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        project.transfer_project(group.id)
 
     project = gl.projects.get(project.id)
     assert project.namespace["path"] == group.full_path
 
     gl.auth()
-    project.transfer_project(gl.user.username)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        project.transfer_project(gl.user.username)
```

Comment:
What if we just use `transfer()` instead which will be the new method?

Reply:
Can do. I thought you were intentionally testing the deprecated version.

But yeah works for me to just use the new one.

Reply:
I was thinking of that initially but ended up writing a unit test to test the deprecation instead. Thanks ! :)